Mimble Wimble.md

```diff
@@ -0,0 +1,124 @@
+
+Confidential transactions that obscure the amount transferred.
+
+Reduced data and processing requirements for new participants, scaling with the system's current state rather than its entire transaction history.
+
+
+## How does MimbleWimble work?
+
+MimbleWimble can be described as a tool for making efficient and private cryptocurrency transactions. Imagine you're sending secret messages that can only be opened by someone with a specific key. In MimbleWimble, when you send cryptocurrency, the amount you send is hidden using a secret code (called a "blinding factor"). This means only you and the person receiving the money know how much is sent.
+
+Even though the amount is secret, everyone can still ensure the transaction is fair (no extra money out of thin air) by checking that what's sent equals what's received, without actually seeing the amounts. This is like knowing a box is full of apples without knowing how many apples are inside.
+
+Transactions in MimbleWimble can be combined in a way that makes it hard for outsiders to figure out who sent money to whom. This is similar to mixing several messages so that no one can tell which message came from which sender. This mixing helps make transactions even more private.
+
+However, MimbleWimble can't do some of the tricks that other protocols can, like creating special conditions for transactions or swapping currencies directly without a middleman. But it has a cool trick: as transactions are combined and old, unnecessary data is removed, stopping the history of transactions from growing too big. This keeps things running smoothly without needing a super-powerful computer.
+
+(Animated video explaining Mimblewimble)
+https://www.youtube.com/watch?v=aIh0XjhNgBc
+
+## Elliptic Curve Cryptography (ECC)
+
+Talking a little about the technical part, MimbleWimble uses a form of cryptography known as Elliptic Curve Cryptography (ECC). ECC enables the protocol to confirm the accuracy of transaction amounts and the identities of the sender and receiver, all while keeping this information private. The "discrete" aspect in logarithms, which ECC is based upon, refers to a mathematical field focused on distinct values, incorporating areas like probability and set theory, thereby enhancing the protocol's security.
+
+With the use of the Elliptic Curve Cryptography, Mimblewimble can achieve 3 main goals for its users:
+
+Privacy Protection: MimbleWimble ensures that the transaction details remain a secret between the sender and the receiver. Only they can know how much crypto was sent.
+
+Transaction Legitimacy: It verifies that transactions are valid without the need for those to be signed publicly, maintaining user anonymity.
+
+Enhanced Efficiency:  The protocol achieves greater efficiency and scalability by generating smaller blocks of data. This approach prevents the blockchain from becoming excessively large and difficult to manage over time, a challenge often faced by networks like Bitcoin.
```

Comment:
"greater efficiency and scalability by generating smaller blocks of data" This statement isn't very informative. 

For instance, presumably, we are talking about a single or multiple transactions. In this case, what is the size without MimbleWimble and what is the smaller size with MimbleWimble? 

Finding out this information will give users a clear understanding of how much more efficient and scalable the protocol is. This information should be available on the Litecoin chain.